OpenTabletDriver.UX/Tools/ReportFormatter.cs

```diff
@@ -117,6 +126,21 @@ private static IEnumerable<string> GetStringFormat(ITouchReport touchReport)
                     yield return touch.ToString();
         }
 
+        private static IEnumerable<string> GetStringFormat(IAbsoluteWheelReport wheelReport)
+        {
+            yield return $"Wheel:{wheelReport.Position?.ToString() ?? "Idle"}";
+            yield return $"Wheel Delta:{wheelReport.Delta?.ToString() ?? "Idle"}";
```

Comment:
just don't return anything if they don't exist, imo
```suggestion
            if (wheelReport.Position != null)
                yield return $"Wheel:{wheelReport.Position.ToString()}"
            if (wheelReport.Delta != null)
                yield return $"Wheel Delta:{wheelReport.Delta.ToString()}";
```

Reply:
It would be harder to see if a Wheel report is actually happening.
(Especially since with the current design, if a position is reported, Delta will be defined)

Reply:
we can try keeping it in and if it ends up being confusing on some tablets we can take it from there